.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,40 @@
+---
+name: 🐞 Bug
+about: File a bug report
+title: '[BUG] <title>'
+labels: bug, triage
+assignees: ''
+
+---
+
+<!--
+Note: Please search to see if an issue already exists for the bug you encountered.
+-->
+
+### Current Behavior:
+<!-- A concise description of what you're experiencing. -->
+
+### Expected Behavior:
+<!-- A concise description of what you expected to happen. -->
+
+### Steps To Reproduce:
+<!--
+Example: steps to reproduce the behavior:
+1. In this environment...
```

Comment:
Should this be removed since environment is also its own section below or should the environment details go here?

Reply:
This is just the standard template github recommends, so I went with it. I think its fine either way so I am inclined to leave it unless you have a strong preference.

Reply:
No strong preference, leaving it is good with me